### ISSUE TITLE
24638 has weird sky fit on r3; bad trace fit

### ISSUE THREAD

**Issue by schlafly** [desihub/desisurveyops#45]
No obvious impact on QA, but the sframes look pretty bad on r3.
https://data.desi.lbl.gov/desi/spectro/redux/daily/tiles/cumulative/24638/20220329/tile-qa-24638-thru20220329.png

![image](https://user-images.githubusercontent.com/1417841/164815365-0887201f-cbb8-476d-8281-897769c76716.png)
https://data.desi.lbl.gov/desi/users/raichoor/main-status/sframesky/sframesky-20220329-00127995.png

**Comment by sbailey** [comment]
@julienguy could you check this case and whether it is a "runaway PCA correction" or other problem?

**Comment by julienguy** [comment]
This is caused by a (rare) trace shift fit on sky lines that went wrong.
```
 plot_fiber_traces --inpsf /global/cfs/cdirs/desi/spectro/redux/daily/exposures/20220329/00127995/psf-r3-00127995.fits --other-psf /global/cfs/cdirs/desi/spectro/redux/daily/calibnight/20220329/psfnight-r3-20220329.fits
```
![Figure_2](https://user-images.githubusercontent.com/5192160/185455782-0b222399-a69e-42bb-b8e1-6c6669f091d8.png)

**Comment by julienguy** [comment]
The wavelength calibration adjustment on sky lines can be improved with a median filtering of the spectra. This takes a bit more time but more importantly it should be tested on a large data sample (with for instance radial velocities comparison).  This is studied in https://github.com/desihub/desispec/issues/1827.

**Comment by schlafly** [comment]
@akremin, can we mark r3 on 20220329/127995, tileid 24638 as bad and reprocess?

**Comment by akremin** [comment]
Since this was a single exposure the standard star fitting fails without r3, so I've flagged the entire petal as bad for this exposure and reprocessed.

Specifically, I've flagged a3 for (TILEID 24638, NIGHT 20220329, EXP 127995). for both the svn `exposure_tables` and the daily versions (which are a snapshot of the daily processed data). I've removed the old reductions and rerun. I have also updated the tsnr table (`tiles-daily.csv`) for that night, as well as running `desi_night_qa` with `--recompute`.
